src/status_im2/contexts/activity_center/notification/common/style.cljs

```diff
@@ -1,8 +1,56 @@
 (ns status-im2.contexts.activity-center.notification.common.style
   (:require [quo2.foundations.colors :as colors]))
 
+(def swipe-action-width 72)
+(def swipe-button-border-radius 16)
+
 (def user-avatar-tag
   {:background-color colors/white-opa-10})
 
 (def user-avatar-tag-text
   {:color colors/white})
+
+(def left-swipe-opacity-interpolation
```

Comment:
probably better to add `-js` suffix to the name ?

Reply:
Good idea, will do it.